docs/source/install.md

```diff
@@ -0,0 +1,85 @@
+# Install Fabric and Fabric Samples
+
+Please install the [Prerequisites](./prereqs.html) before following these install instructions.
+
+We think the best way to understand something is to use it yourself.  To help you use Fabric, we have created a simple Fabric Network called `test-network` using docker compose, and a set of sample applications that demonstrate its core capabilities.
```

Comment:
Again, "Fabric network" is fine. This is not the proper name of a particular network.

Reply:
done.